src/discord/commands/manage.ts

```diff
@@ -0,0 +1,166 @@
+import Timeblock from '../../models/timeBlock.model';
+
+function updateEmbed(userArray: string[], maxCapacity: number, manageState: string) {
+    const embedTitle = manageState === 'add' ? 'Adding' : 'Removing';
+    const preposition = manageState === 'add' ? 'to' : 'from';
+
+    const embed = new MessageEmbed()
+        .setColor('#48d7fb')
+        .setTitle(`Currently ${embedTitle} Collaborators`)
+        .setDescription(
+            `To ${manageState} collaborators ${preposition} your booking, @ their Discord username!\n\nCurrently Invited Collaborators (${
+                maxCapacity! - userArray!.length
+            } available spaces left):\n<@!${userArray?.join('>\n<@!')}>`
+        );
+
+    return embed;
+}
+
+function updateButtons(manageState: string) {
+    const buttonRow = [new MessageButton().setCustomId('completeBooking').setLabel('Complete Booking').setStyle('PRIMARY')];
+    const label = manageState === 'add' ? 'Remove' : 'Add';
+    const style = manageState === 'add' ? 'DANGER' : 'SUCCESS';
+
+    buttonRow.push(new MessageButton().setCustomId(`${label.toLowerCase()}Collaborators`).setLabel(`${label} Collaborators`).setStyle(style));
+
+    return new MessageActionRow().addComponents(buttonRow);
+}
+
+async function mongoDBFilter(userArray: string[], bookingId: string) {
+    //Ensures no duplicate members are listed under users of a certain timeblock (might need to alter this after updates)
+    const filterArray = [];
+    const currentUserArray = (await Timeblock.findOne({ _id: bookingId }))!.users;
+
+    for (const user of userArray) {
+        if (!currentUserArray.includes(user)) {
+            filterArray.push(user);
+        }
+    }
+    return filterArray;
+}
+
+async function parseCommandOptions(interaction: CommandInteraction) {
+    const bookingId = interaction.options.getString('booking-id');
+    const bookedTimeblock = await Timeblock.findOne({ _id: bookingId });
+
+    if (bookedTimeblock === null) {
+        interaction.reply({ content: 'Invalid Booking ID: Do `/bookings` to see all your current bookings!', ephemeral: true });
+        return undefined;
+    }
+
+    //TODO: Finish This after updates to the timeblock model
+}
+
+export default {
+    name: 'manage',
+    description: 'Manage Collaborators in Booking!',
+    options: [
+        {
+            type: 3,
+            name: 'booking-id',
+            description: 'ID of your booking',
+            required: true,
+        },
+        {
+            type: 3,
+            name: 'add-or-remove',
+            description: 'Add or remove collaborators',
+            choices: [
+                {
+                    name: 'add',
+                    value: 'add',
+                },
+                {
+                    name: 'remove',
+                    value: 'remove',
+                },
+            ],
+            required: true,
+        },
+        {
+            type: 3,
```

Comment:
Should this be type 5 (`USER`)? It might be good to add a few extra optional fields so that you can add/remove multiple at a time.